src/Collectors/CommandCollector.php

```diff
@@ -75,7 +74,7 @@ protected function send($event): void
             $this->agent->send();
         } catch (ClientException $exception) {
             Log::error($exception, ['api_response' => (string) $exception->getResponse()->getBody()]);
-        } catch (Throwable $t) {
+        } catch (\Throwable $t) {
```

Comment:
This looked better before -- is this why the linter complained? Very, very weird.

Reply:
Yes, we were a few versions behind and looks like they [change the default](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/6662) a couple of weeks ago.

Reply:
They didn't much of an explanation after I followed the thread to [this comment](https://github.com/symfony/symfony/pull/47709#discussion_r982303782). Must be a Symfony thing.